nginx-controller/nginx/configurator.go

```diff
@@ -154,32 +204,34 @@ func (cnf *Configurator) generateNginxCfg(ingEx *IngressEx, pems map[string]stri
 			server.JWTLoginURL = ingCfg.JWTLoginURL
 		}
 
-		var locations []Location
-		rootLocation := false
+		if ingEx.MergeableType != "master" {
```

Comment:
Does this mean that Master cannot have locations?

Reply:
Maybe it makes sense for the master to handle the `/` location. This means it will handle all the requests for which there is no path in the Ingress resource.